### ISSUE TITLE
Zoomed topo should start zoomed

### ISSUE THREAD

**Issue by birgander2** [theCrag/tca#41]
When clicking the zoom button on a topo, a new page appears with almost the same zoom level. It should start with (at least) a zoom level to fill all the vertical space and no black bars on top and bottom.

**Comment by bkucsera** [comment]
When you click on an image to enlarge, the result is depending on the orientation of the image (landscape or portrait).
If we zoom in, we can "loose" the two sides of the image. Isn't it more disturbing as a user? Or you mean something else?

**Comment by birgander2** [comment]
No, exactly this. I think it's better to zoom in until any black border is visible. Like that, I potentially only have to scroll around a bit. In the current way, I typically first have to zoom and then scroll, too.

**Comment by bkucsera** [comment]
I have tested this with several topo images and i think zooming in would be more confusing for users. The standard solution in mobile apps is the way we have it now. I suggest to keep it this way and we can check later if we get more comments on this from a wider audience.
